app/views/layouts/_google_analytics.html.erb

```diff
@@ -1,12 +1,11 @@
 <% if ENV['GOOGLE_ANALYTICS_ID'].present? %>
+<!-- Global Site Tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-51169916-1"></script>
```

Comment:
This should use the env var not the hard coded id.

Reply:
good catch! updated